src/components/panel/panel.tsx

```diff
@@ -381,19 +407,18 @@ export class Panel implements ConditionalSlotComponent, InteractiveComponent {
   }
 
   renderHeaderActionsEnd(): VNode {
-    const { dismiss, dismissible, el, intlClose, closable } = this;
+    const { dismiss, el, intlClose, closable } = this;
     const text = intlClose || TEXT.close;
 
-    const dismissibleNode =
-      closable || dismissible ? (
-        <calcite-action
-          aria-label={text}
-          icon={ICONS.close}
-          onClick={dismiss}
-          ref={this.setDismissRef}
-          text={text}
-        />
-      ) : null;
+    const dismissibleNode = closable ? (
```

Comment:
nitpick: Maybe we should rename this to `closeNode`

Reply:
Do we have to change `setDismissRef` to `setCloseRef` and  `dismissButtonEl` to `closeButtonEl` . thoughts?

Reply:
That would be nice too.